.github/workflows/docker.yml

```diff
@@ -31,6 +36,16 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      # Install the cosign tool except on PR
+      # https://github.com/sigstore/cosign-installer
+      - name: Install cosign
```

Comment:
Why exactly do we have to manually install cosign? https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/container#generating-provenance looks like we basically just have to add the `provenance` step and are done with it?

Reply:
Yeah, I had 2 changes as part of this MR actually.
1. The container signing and 
2. The other was the SLSA provenance. 

I will remove the container signing part. That seems like a separate issue.